app/code/Magento/Catalog/view/adminhtml/templates/catalog/product/attribute/js.phtml

```diff
@@ -196,9 +196,11 @@ function switchDefaultValueField()
 
         <?php foreach ($this->helper('Magento\Catalog\Helper\Data')->getAttributeHiddenFields() as $type => $fields): ?>
             case '<?= /* @escapeNotVerified */ $type ?>':
+                var is_front_tab_hidden = false;
```

Comment:
Could you change this variable to `isFrontTabHidden` to match the rest of the file.